src/main/java/com/elasticbox/jenkins/auth/TokenAuthentication.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 ElasticBox
+ *
+ * Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or http://apache.org/licenses/LICENSE-2.0>
+ * or the MIT license <LICENSE-MIT or http://opensource.org/licenses/MIT> , at your option.
+ * This file may not be copied, modified, or distributed except according to those terms.
+ */
+
+package com.elasticbox.jenkins.auth;
+
+public class TokenAuthentication implements Authentication {
```

Comment:
Before there were more methods because there were several authentication modes. 
Please, keep it while username and password mode is kept or discarded